chore: update readme (#78)
Informasjonsforvaltning/catalog-admin-service: README.md

# catalog-admin-service

> This service is responsible for settings in concept catalogs. This includes code lists, internal fields and editable fields. It is also responsible for settings across different catalogs including design and responsible users. 

Reviewer: ```suggestion
This service is responsible for settings in registration catalogs. This includes code lists, internal fields and editable fields for concepts, and settings across different catalogs including design and responsible users. 
```

This is a REST API. The Open-API spec can be found [here](https://raw.githubusercontent.com/Informasjonsforvaltning/catalog-admin-service/main/openapi.yaml). 

## Requirements
- maven
- java 21
- docker
- docker-compose

## Run tests
```
mvn test
```

## Run locally

### Start catalog admin service
Start catalog admin service locally using maven. Use Spring profile **develop**.
```
mvn spring-boot:run -Dspring-boot.run.profiles=develop
```

## Usage examples

> Obs! Remember to use correct catalogId. 

Reviewer: Blir ikke `Obs` veldig norsk, hva med `Note`?
```suggestion
Note! Remember to use correct catalogId. 
```
